api/src/pcapi/core/bookings/external/booking_notifications.py

```diff
@@ -26,18 +27,19 @@ def send_today_events_notifications_metropolitan_france() -> None:
     France but not the morning (11h UTC -> 12h/13h local time) and
     send notifications to all the users.
     """
-    today_min, today_max = _setup_today_min_max(utc_mean_offset=1)
-    stock_ids = find_today_event_stock_ids_metropolitan_france(today_min, today_max)
+    if not FeatureToggle.WIP_DISABLE_TODAY_STOCK_NOTIFICATION.is_active():
```

Comment:
Suggestion : on préfère en général le `return` rapide plutôt qu'encapsuler dans un bloc : 
```suggestion
    if FeatureToggle.WIP_DISABLE_TODAY_STOCK_NOTIFICATION.is_active():
        return
```
Idem pour les suivants.